packages/ActionMenu/src/ActionMenu.test.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import MockDropdown from '@igloo-ui/dropdown/__mocks__/dropdownMock';
+import ActionMenu, { ActionMenuOption } from './ActionMenu';
+
+const textOnlyList: ActionMenuOption[] = [
+  {
+    label: 'Add Item',
+    value: 'add',
+  },
+  {
+    label: 'Delete Item',
+    value: 'delete',
+    disabled: true,
+  },
+  {
+    label: 'Copy Item',
+    value: 'copy',
+  },
+];
+
+const setup = (props = { options: textOnlyList }) => {
+  return render(
+    <ActionMenu dataTest="ids-action-menu" {...props}>
+      Hello world
+    </ActionMenu>
+  );
+};
+
+jest.mock('@igloo-ui/dropdown', () => ({
+  __esModule: true,
+  default: jest.fn(MockDropdown),
+}));
+
+describe('ActionMenu', () => {
```

Comment:
I would like to start increasing our test coverage, can you add a test to validate that the `Option` in the list are displayed and another for `onOptionSelect`